firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -101,6 +120,46 @@ export class FirestoreBigQueryEventHistoryTracker
     return rows;
   }
 
+  getTimePartitionParameterField(data, documentName) {
+    const fieldName = this.config.timePartitioningField;
```

Comment:
Do we check to ensure all of these are available? What happens if `fieldType` for example is missing?